riverdata/App.js

```diff
@@ -22,6 +23,7 @@ function App() {
         <Stack.Screen name="Site Gauges" component={RDSiteGaugesScreen} />
         <Stack.Screen name="Gauge Graph" component={RDGaugeGraphScreen} />
       </Stack.Navigator>
+      <Tabs />
```

Comment:
@HenryJames4 See https://reactnavigation.org/docs/nesting-navigators

It looks like having a Tab controller inside of a navigation stack is a no-no.
Here's the error I get when running `npm run ios`

```
ERROR  Error: Another navigator is already registered for this container. You likely have multiple navigators under a single "NavigationContainer" or "Screen". Make sure each navigator is under a separate "Screen" container. See https://reactnavigation.org/docs/nesting-navigators for a guide on nesting.

This error is located at:
    in BottomTabNavigator (created by Tabs)
    in Tabs (created by App)
    in EnsureSingleNavigator
    in BaseNavigationContainer
    in ThemeProvider
    in NavigationContainerInner (created by App)
    in App (created by ExpoRoot)
    in ExpoRoot
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in main(RootComponent)
```